### DISCUSSION TITLE
i think these are supposed to be mm & m/s

### DISCUSSION THREAD

**Discussion by nerdCopter** [supermerill/SuperSlicer#2041] [Q&A] [answered]
![wrong-measurement](https://user-images.githubusercontent.com/56646290/145684088-dcafc7c9-82bf-41de-8685-9f7662ebd8e3.png)
&
one of those seems correct with `m/s`, but not the other.
`detraction`, not `derectraction`.

**Comment by supermerill** [comment]
can't reproduce. what version?
Please open an issue if you found a bug.

I think i'm the only one that can delete it.

**Reply by nerdCopter** [reply]
> can't reproduce. what version? Please open an issue if you found a bug.
> 
> I think i'm the only one that can delete it.

`SuperSlicer-ubuntu_18.04-2.3.57.6.AppImage`
maybe it's a gtk2 vs 3 problem  :man_shrugging:

**Comment by nerdCopter** [comment, marked as answer]
`SuperSlicer-ubuntu_18.04-gtk2-2.3.57.6.AppImage` renders proper.  it's a gtk3 issue, but maybe not a superslicer issue.
